### PR TITLE
Vg 37 make pages more representable

### PR DESCRIPTION
**Pull request by marcusju23**
Closes related issue. 

This PR add:
- Styling to homepage
- Styling to product detail page
- General styling
- Solves https://github.com/marcusju23/vueproject-group7/pull/26#pullrequestreview-2357169092

### CODE REVIEW

**Review by Danback — DISMISSED** [review]
Looks good to me, good job!

**Comment by marcusju23** [comment]
Addressing https://github.com/marcusju23/vueproject-group7/pull/26#pullrequestreview-2357169092

File: src/assets/style.css [hunk 1]
```diff
@@ -2,28 +2,62 @@
...
+}
+
 .active {
-    font-weight: bold;
-    color: beige;
+    @apply font-bold text-amber-100;
 }
 
 .menu-item {
-    @apply block px-4 py-2 hover:bg-gray-700;
```
Comment (Danback) [inline src/assets/style.css:11]: This doesn't seem to work

**Review by Danback — DISMISSED** [review]
I think everything looks good! Some styling on the products was lost but it was expected having to change to tailwind (my mistake) and we can just simply add this again. Great job, impressive fix!

File: src/assets/style.css (hunk 1 above)
Comment (marcusju23) [reply src/assets/style.css:11]: What is not working exactly?

**Comment by marcusju23** [comment]
Closed and reopened PR because of bug.

**Review by Danback — APPROVED** [review]
Nice

**Review by Demppas — APPROVED** [review]
allt ser bra ut!

**Pull request merged**